styles.css

```diff
@@ -1 +1,73 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+* {
+    padding: 0;
+    margin: 0;
+}
+
+.container {
+    background-color: #c0cbc3;
+    padding: 16px;
+    color: #463e4a;
+
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 50px auto 50px 50px 50px 50px 50px;
+    column-gap: 16px;
+    row-gap: 16px;
+    grid-template-areas:
+        "header"
+        "text"
+        "section-c"
+        "section-d"
+        "section-e"
+        "none"
+        "footer"
+}
+
```

Comment:
Dit is inderdaad handig: de meeste mensen zullen wel op mobiel kijken ipv desktop